### ISSUE TITLE
Disable same origin policy in Chrome

### ISSUE THREAD

**Issue by MANDA-LAWINE** [responsively-org/responsively-app#566]
Is there a way to disable same origin policy ? 
In chrome we can do that, see [disable-same-origin-policy-in-chrome](https://stackoverflow.com/questions/3102819/disable-same-origin-policy-in-chrome)

**Comment by jjavierdguezas** [comment]
Based on what you linked, I think it could be done toggling [`<webview>`'s `disablewebsecurity` attribute]( https://www.electronjs.org/docs/api/webview-tag#disablewebsecurity)

**Comment by mahendra1290** [comment]
Hi @jjavierdguezas i am new to open source and worked on ReactJs and Angular can I try to solve this issue.

**Comment by jjavierdguezas** [comment]
> Hi @jjavierdguezas i am new to open source and worked on ReactJs and Angular can I try to solve this issue.

of course you can! thanks!

**Comment by dmzinc** [comment]
Hi @jjavierdguezas I am new to open source contribution and I would love to know how I can contribute

**Comment by mahendra1290** [comment]
@jjavierdguezas Did it need to be a command line argument ?

**Comment by jjavierdguezas** [comment]
> @jjavierdguezas Did it need to be a command line argument ?

I don't think so, it should be a toggle in the UI that toggles the webview attribute

**Comment by mahendra1290** [comment]
> > @jjavierdguezas Did it need to be a command line argument ?
> 
> I don't think so, it should be a checkbox in the UI that toggles the webview attribute cc: @manojVivek

Ok i understood can u guide me on it where I should add the toggle option.

**Comment by mahendra1290** [comment]
there are two npm scripts start-main-dev and start-renderer-dev what's the difference

**Comment by jjavierdguezas** [comment]
> > > @jjavierdguezas Did it need to be a command line argument ?
> > 
> > 
> > I don't think so, it should be a checkbox in the UI that toggles the webview attribute cc: @manojVivek
> 
> Ok i understood can u guide me on it where I should add the toggle option.

![image](https://user-images.githubusercontent.com/13673443/114592519-524b8400-9c8b-11eb-82fc-2fbb8e7175df.png)

**Comment by mahendra1290** [comment]
ya i have done that

**Comment by mahendra1290** [comment]
> Based on what you linked, I think it could be done by toggling [`<webview>`'s `disablewebsecurity` attribute](https://www.electronjs.org/docs/api/webview-tag#disablewebsecurity)

I tried this but it is not working.
`<webview
          ref={this.webviewRef}
          preload="./preload.js"
          className={cx(styles.device)}
          src={address || 'about:blank'}
          useragent={useragent}
          style={deviceStyles}
          disablewebsecurity={true}
        />`

**Comment by shanemcfadden** [comment]
Hi there! Is this issue up for grabs? I'm looking for a way to start contributing.

**Comment by jjavierdguezas** [comment]
> Hi there! Is this issue up for grabs? I'm looking for a way to start contributing.

 I think @mahendra1290 is on it
